<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ircontrolwindow</class>
 <widget class="QMainWindow" name="ircontrolwindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ir control</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tab_layout">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="serial">
       <attribute name="title">
        <string>serial</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QComboBox" name="ser_dropdown"/>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="ser_con_button">
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>connect</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="ser_refresh_button">
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>refresh</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="ser_discon_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>disconnect</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="binds">
       <attribute name="title">
        <string>binds</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="2" column="0">
         <widget class="QTableWidget" name="bind_table">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="3">
           <widget class="QLabel" name="command_label">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>command</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLineEdit" name="command_input"/>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="bind_input"/>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="bind_label">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>bind</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="argument_label">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>arguments</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QLineEdit" name="argument_input"/>
          </item>
          <item row="2" column="5">
           <widget class="QPushButton" name="bind_add_button">
            <property name="text">
             <string>add</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>bind_input</tabstop>
  <tabstop>command_input</tabstop>
  <tabstop>argument_input</tabstop>
  <tabstop>bind_add_button</tabstop>
  <tabstop>ser_con_button</tabstop>
  <tabstop>ser_refresh_button</tabstop>
  <tabstop>tab_layout</tabstop>
  <tabstop>bind_table</tabstop>
  <tabstop>ser_discon_button</tabstop>
  <tabstop>ser_dropdown</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
